### PR TITLE
Add blogpost waterinfo-tidal-eel

### PR DESCRIPTION
**Pull request by stijnvanhoey**
Hi @stefaniebutland here's (finally) our blog post on the [wateRinfo package](https://github.com/ropensci/wateRinfo). We opted to introduce the package using a real world case about the tidal behaviour of eel. Suggestions to improve the text are very welcome.

### CODE REVIEW

**Comment by stijnvanhoey** [comment]
Apparently, the deployment failed due to the authors section I orignally added, I'll add an author's index as explained in the readme.

**Comment by stefaniebutland** [comment]
Thank you @stijnvanhoey. Big smile on my face as I see the smiling eel again in the preview: https://deploy-preview-403--ropensci.netlify.com/blog/2019/01/15/waterinfo-tidal-eel/

I will review and get you feedback later today.

**Comment by peterdesmet** [comment]
👋 @stefaniebutland two remarks regarding layout:

- It looks like footnotes are not completely supported: they do end up at the bottom, but reference in text is broken
- Tables go beyond width of post

File: content/blog/2019-01-15-waterinfo-tidal-eel.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+>
+> *Vizzini, [The Princess Bride](https://youtu.be/KGcat9tGZVU)*
+
+European eels ([*Anguilla anguilla*](https://en.wikipedia.org/wiki/Anguilla_anguilla)) have it tough. Not only are they depicted as monsters in movies, they are [critically endangered](https://doi.org/10.2305/IUCN.UK.2014-1.RLTS.T60344A45833138.en) in real life. One of the many aspects that is contributing to their decline is the reduced connectivity between their freshwater and marine habitats. Eels are catadromous: they live in freshwater, but migrate to the Sargasso Sea to spawn, a route that is blocked by numerous human structures (shipping locks, sluices, pumping stations, etc.). [Pieterjan Verhelst](http://www.marinebiology.ugent.be/node/68491) studies the impact of these structures on the behaviour of eels, making use of the [fish acoustic receiver network](http://lifewatch.be/en/fish-acoustic-receiver-network) that was established as part of the Belgian LifeWatch observatory. This animated video gives a quick introduction to his research and the receiver network:
+
+[![Tagging research on eel in Belgium](https://img.youtube.com/vi/7YQVgl3QPyY/0.jpg)](https://www.youtube.com/watch?v=7YQVgl3QPyY)
```
Comment (stefaniebutland) [inline content/blog/2019-01-15-waterinfo-tidal-eel.md]: Can you add some of this credit text for the video? "Tagging research on eel in Belgium. Animated video. Coordination: Karen Rappé and Pieterjan Verhelst, Animation Steve Bridger, Voice: Bryan Kopta (EN), Production: LifeWatch Belgium, Executive producer: VLIZ Oostende -  Belgium."

File: content/blog/2019-01-15-waterinfo-tidal-eel.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+In this blog post, we'll explore if the migration of one eel is influenced by the tide. It's a research use case for our R package [wateRinfo](https://ropensci.github.io/wateRinfo/), which was recently [peer reviewed](https://github.com/ropensci/software-review/issues/255) (thanks to reviewer [Laura DeCicco](https://github.com/ldecicco-USGS) and editor [Karthik Ram](https://github.com/karthik) for their constructive feedback!) and accepted as a community-contributed package to [rOpenSci](https://github.com/ropensci/wateRinfo).
```
Comment (stefaniebutland) [inline content/blog/2019-01-15-waterinfo-tidal-eel.md]: please put `wateRinfo` in backticks in the post

File: content/blog/2019-01-15-waterinfo-tidal-eel.md (hunk 1 above)
Comment (stijnvanhoey) [reply content/blog/2019-01-15-waterinfo-tidal-eel.md]: Good point, I'll add this to the acknowledgments section.

File: content/blog/2019-01-15-waterinfo-tidal-eel.md (hunk 2 above)
Comment (stijnvanhoey) [reply content/blog/2019-01-15-waterinfo-tidal-eel.md]: wateRinfo -> `wateRinfo` done

File: content/blog/2019-01-15-waterinfo-tidal-eel.md (hunk 1 above)
Comment (stijnvanhoey) [reply content/blog/2019-01-15-waterinfo-tidal-eel.md]: @peterdesmet can you have a quick check on the adapted acknowledgements?

File: content/blog/2019-01-22-waterinfo-tidal-eel.md [hunk 3]
```diff
@@ -0,0 +1,176 @@
+---
+slug: "waterinfo-tidal-eel"
+title: wateRinfo - Downloading tidal data to understand the behaviour of a migrating eel
+authors:
+  - Stijn Van Hoey
+  - Peter Desmet
+date: 2019-01-15
```
Comment (stefaniebutland) [inline content/blog/2019-01-22-waterinfo-tidal-eel.md]:
```suggestion
date: 2019-01-22
```

File: content/blog/2019-01-15-waterinfo-tidal-eel.md (hunk 1 above)
Comment (peterdesmet) [reply content/blog/2019-01-15-waterinfo-tidal-eel.md]: Updated.

**Comment by peterdesmet** [comment]
@stefaniebutland all changes have been done. The 3 twitter handles (@SVanHoey @peterdesmet @LifeWatchINBO) you mention are ok!

**Comment by stefaniebutland** [comment]
This is great thank you!

**Comment by stijnvanhoey** [comment]
That is certainly ok, thanks for noticing.

**Pull request merged**